src/layouts/sidebar-sidecar/server.ts

```diff
@@ -200,6 +201,26 @@ export function getStaticGenerationFunctions<
         basePath,
       ])
 
+      /**
+       * Figure out of a specific docs version is being viewed
+       *
+       * @TODO `indexOfVersionPathPart` will be used in quick-follow PR
+       */
+      // let indexOfVersionPathPart
+      let versionPathPart
+      if (versions) {
+        pathParts.find((pathPart, index) => {
+          const matchingVersion = versions.find(
+            (version) => pathPart === version.version
+          )
+          if (matchingVersion) {
+            versionPathPart = pathPart
+            // indexOfVersionPathPart = index
```

Comment:
⛏️  Can we remove this comment? Or is that done in one of the follow-ups?

Reply:
It's done in one of the follow-ups, I noted this in the TODO above this section 😊

Reply:
👀  Sure enough!